app/about_us/OurCrew.js

```diff
@@ -0,0 +1,88 @@
+import "./OurCrew.css";
+
+const OurCrew = () => {
+  return (
+    <section className="crew-section">
```

Comment:
Instead of hardcoding each crew member, you can create the crew members data in an array and map over it.